src/plait/core.clj

```diff
@@ -1,5 +1,6 @@
 (ns plait.core
-  (:require [clojure.walk :as w]))
+  (:require [clojure.walk :as w]
+            [clojure.walk :as walk]))
```

Comment:
[Reminds me of little miss twins](https://www.google.com/search?channel=fs&client=ubuntu-sn&q=little+miss+twins)